tests/proxysql-admin-testsuite.bats

```diff
@@ -121,13 +121,24 @@ fi
 }
 
 
+@test "run proxysql-admin --update-mysql-version ($WSREP_CLUSTER_NAME)" {
+  run sudo PATH=$WORKDIR:$PATH $WORKDIR/proxysql-admin --update-mysql-version
+  echo "$output" >&2
+  [ "$status" -eq  0 ]
+}
+
+@test "run the check for --update-mysql-version ($WSREP_CLUSTER_NAME)" {
+  local check_mysql_version=$(mysql_exec "$HOST_IP" "$PORT_3" "SELECT VERSION();" | tail -1 | cut -d'-' -f1)
```

Comment:
For the test, may want to set mysql-server_version to a known value, say "0.0.0" to be sure that we test it gets changed.

Reply:
Since we are checking the version between ProxySQL DB and MySQL Cluster do we really need to test with known value?